src/components/templates/CreateTeam.jsx

```diff
@@ -23,7 +26,7 @@ const CreateTeam = () => {
   const { userId } = useSelector(state => state.currentUser);
   const [teams, fetchTeams] = useTeams(id);
   const team = teams.find(t => t.team_lead === userId);
-
+debugger
```

Comment:
Should these `debugger` statements be removed?

Reply:
Will remove it!